### PR TITLE
Add doc for precompiled demos

### PR DESCRIPTION
**Pull request by sloriot**

### CODE REVIEW

**Comment by sloriot** [comment]
[New doc](https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-5.3-Ic-101/output1/Manual/windows.html#sec-win-demo) looks fine.

**Comment by sloriot** [comment]
Strange % in navtree
Screenshot from 2021-04-09 09-35-05

**Comment by sloriot** [comment]
IIRC, this % should have been removed by the post-processing script (nothing to do with my changes)

**Comment by sloriot** [comment]
Nevermind it is only with 1.8.4

File: Documentation/doc/Documentation/windows.txt [hunk 1]
```diff
@@ -279,4 +279,11 @@ you are advised to look at the `CMakeLists.txt` files in the example
 folder of the package(s) that you are using to learn how to specify \cgal and additional third party
 dependencies.
 
+\section sec-win-demo Trying CGAL algorithms using a Precompiled Demo
+\cgal provides a few demos providing a graphical user interfaces for some \cgal algorithms.
+All precompiled demos are available for download in the \ref packages page.
```
Comment (MaelRL) [inline Documentation/doc/Documentation/windows.txt:284]:
```suggestion
\cgal provides a few demos offering a graphical user interface for some \cgal algorithms.
All precompiled demos are available for download on the \ref packages page.
```

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.3-Ic-106.shtml

**Pull request merged**